plugins/carbon_black_response/unit_test/test_uninstall_sensor.py

```diff
@@ -11,8 +11,6 @@
 post_uninstall = {
     "systemvolume_total_size": "42947571712",
     "emet_telemetry_path": "",
-    "os_environment_display_string": "Windows 10 Server Enterprise (Evaluation), 64-bit",
-    "emet_version": "",
```

Comment:
why did we remove this ? I can't remember exactly why, was it not being used?

Reply:
It wasn't being used I believe?